app/src/test/java/org/simple/clinic/contactpatient/ContactPatientCallPatientUiRendererTest.kt

```diff
@@ -21,7 +20,7 @@ import org.threeten.bp.LocalDate
 import org.threeten.bp.Period
 import java.util.UUID
 
-class ContactPatientUiRendererTest {
+class ContactPatientCallPatientUiRendererTest {
```

Comment:
I am following the convention of the rest of the codebase where the UI renderer is prefixed with the feature name. Maybe we should decide once and for all if we should have the prefix for the class name or nor?

My two cents is that we should because if we look at a test report, the names of the classes will all be `UiRendererTest`.

Reply:
I agree that we should have the feature names as prefix. I am just asking to remove the `ContactPatient` prefix since the package name and `CallPatient` is sufficient to point that this is a feature within `ContactPatient`.

Reply:
Ah, for this specific feature I think it makes sense. I will do this.